apps/chat/src/utils/app/file.ts

```diff
@@ -361,3 +367,6 @@ export const getNextFileName = (
 
   return `${prefix}${maxNumber + 1}${defaultFileExtension}`;
 };
+
+export const prepareFileName = (filename: string) =>
```

Comment:
Do we really need this? Why we can't just use filename?

Reply:
@Alexander-Kezik , the reason is described in PR description